src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -395,6 +409,10 @@ public String getCcdAccessControlCaseworkerRoleRegex() {
         return ccdAccessControlCaseworkerRoleRegex;
     }
 
+    public boolean getCcdNewAccessControlEnabled() {
```

Comment:
it's good practice always having getter method names same as the corresponding property